pom.xml

```diff
@@ -1,389 +1,414 @@
-                    <artifactId>formatter-maven-plugin</artifactId>
-                    <version>${formatter-maven-plugin.version}</version>
-                    <configuration>
-                        <configFile>${project.root.basedir}/eclipse/formatter.xml
-                        </configFile>
-                        <lineEnding>CRLF</lineEnding>
-                    </configuration>
-                </plugin>
-                <plugin>
-                    <groupId>com.google.code.maven-license-plugin</groupId>
-                    <artifactId>maven-license-plugin</artifactId>
-                    <version>${com.google.code.maven-license-plugin.version}</version>
-                    <configuration>
-                        <header>${project.root.basedir}/license/header.txt</header>
-                        <includes>
-                            <include>src/**/*.java</include>
-                        </includes>
-                        <encoding>${encoding}</encoding>
-                        <headerDefinitions>
-                            <headerDefinition>${project.root.basedir}/license/header-definition.xml</headerDefinition>
-                        </headerDefinitions>
-                    </configuration>
-                </plugin>
-            </plugins>
-        </pluginManagement>
-    </build>
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>${project.groupId}</groupId>
-                <artifactId>terasoluna-gfw-functionaltest-domain</artifactId>
-                <version>${project.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${project.groupId}</groupId>
-                <artifactId>terasoluna-gfw-functionaltest-web</artifactId>
-                <version>${project.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${project.groupId}</groupId>
-                <artifactId>terasoluna-gfw-functionaltest-env</artifactId>
-                <version>${project.version}</version>
-            </dependency>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>sql-maven-plugin</artifactId>
+          <version>${org.codehaus.mojo.sql-maven-plugin.version}</version>
+        </plugin>
+        <plugin>
+          <groupId>net.revelc.code.formatter</groupId>
+          <artifactId>formatter-maven-plugin</artifactId>
+          <version>${formatter-maven-plugin.version}</version>
+          <configuration>
+            <configFile>${project.root.basedir}/eclipse/formatter.xml
+            </configFile>
+            <lineEnding>CRLF</lineEnding>
+            <directories>
+              <directory>${project.basedir}</directory>
+            </directories>
+            <includes>
+              <include>**/src/main/java/**/*.java</include>
+              <include>**/src/test/java/**/*.java</include>
```

Comment:
OK.
It means `${project.build.sourceDirectory}/java` and `${project.build.testSourceDirectory}/java`.